claat/main.go

```diff
@@ -68,27 +70,34 @@ func main() {
 		os.Exit(1)
 	}
 
+	pm := make(map[string]bool)
```

Comment:
```go
var pm map[string]bool
```

But also, make this whole diff into its own function. Maybe `parsePassMetadata`

Reply:
Done